### DISCUSSION TITLE
Rendering entity in world

### DISCUSSION THREAD

**Discussion by NekiChan** [FabricMC/fabric-api#2643] [Player support]
Game crashes randomly with entity rendering error in version 1.19
here's the crash report , please if there's any way to fix it tell me.

[crash-2022-11-07_23.05.24-client.txt](https://github.com/FabricMC/fabric/files/9954771/crash-2022-11-07_23.05.24-client.txt)

**Comment by warjort** [comment]
Issue with  the continuity mod.

See the big warning text on the mod page.
https://www.curseforge.com/minecraft/mc-mods/continuity

If that doesn't fix it, contact the continuity mod author.
